src/api/api.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+
+const translate = (sourceLang, targetLang, word) => {
+  return axios({
+    method: 'GET',
+    url:
+      'https://systran-systran-platform-for-language-processing-v1.p.rapidapi.com/resources/dictionary/lookup',
+    headers: {
+      'content-type': 'application/octet-stream',
+      'x-rapidapi-host':
+        'systran-systran-platform-for-language-processing-v1.p.rapidapi.com',
+      'x-rapidapi-key': 'bd4714acc3msh8da2c7d829ea9f9p13ab38jsn4b1c796e6687',
```

Comment:
API keys are often (though not always) considered "secrets" that should be kept out of source control. In this case it's probably not a big deal but in general you'll want to ask yourself "What would happen if a malicious actor got a hold of this secret?". The way we usually handle this stuff is through environment variables (See [12 factor config](https://12factor.net/config), [react-native-dotenv](https://github.com/zetachang/react-native-dotenv)).